e2e/claims.service.e2e.ts

```diff
@@ -809,4 +870,118 @@ describe('Сlaim tests', () => {
       ).toBeDefined();
     });
   });
+
+  describe('Verification Tests', () => {
+    const createExampleSignedCredential = async (
+      issuerFields: IssuerFields[],
+      namespace: string,
+      expirationDate?: Date
+    ) => {
+      return await verifiableCredentialsService.createRoleVC({
+        id: rootOwnerDID,
+        namespace: namespace,
+        version: '1',
+        issuerFields,
+        expirationDate,
+      });
+    };
+
+    test('verifyVc should verify a credential with no errors if the Issuer is authorized', async () => {
+      await signerService.connect(rootOwner, ProviderType.PrivateKey);
+      await domainsService.createRole({
+        roleName: verifyVcRole,
+        namespace,
+        data: roles[`${verifyVcRole}.${root}`],
+        returnSteps: false,
+      });
+      const vc = await createExampleSignedCredential(
+        [],
+        `${verifyVcRole}.${root}`
+      );
+      const result = await claimsService.verifyVc(vc);
+      expect(result.errors).toHaveLength(0);
+      expect(result.isVerified).toBeTruthy;
+    });
+    test('verifyVc should: not verify a credential and return an issuer error if the issuer is not authorized', async () => {
+      const roleName = `${verifyVcRole2}.${root}`;
+      await domainsService.createRole({
+        roleName: verifyVcRole2,
+        namespace,
+        data: roles[roleName],
+        returnSteps: false,
+      });
+      const vc = await createExampleSignedCredential([], roleName);
+      const result = await claimsService.verifyVc(vc);
+      expect(result.errors).toHaveLength(1);
+      expect(result.errors).toContain(
+        `Issuer ${rootOwnerDID} is not authorized to issue ${roleName}: issuer is not in DID list`
+      );
+      expect(result.isVerified).toBeFalsy;
+    });
+    test('resolveCredentialAndVerify should resolve a Verifiable Credential and call correct verification method (verifyVC)', async () => {
```

Comment:
The credential is created (I can log it after createExampleSignedCredential), but the resolveCredential method never finds it given a did and subject. Maybe this is expected, but I think if it is created the IPFS resolver should be able to find it?

Reply:
Do I also need to do the createClaimRequest --> issueClaimRequest for this as well, even though I am doing createExampleSignedCredential?